HOW_WE_USE_GITHUB.md

```diff
@@ -238,17 +238,189 @@ please post details to the [Nucleus forums](https://community.anaconda.cloud/).
 
 In order to not have to manually type or copy/paste the above repeatedly, please note that it's possible to add text for the most commonly-used responses via [GitHub's "Add Saved Reply" option][docs-saved-reply].
 
+---
 
-### Types of Tickets
+## Types of issues
 
-#### Standard Ticket/Issue
+The following section describes the different types of issues that we create in our GitHub
+projects and how/why they are used.
+
+## Standard Ticket/Issue
 
 TODO
 
-#### Epics
+## Epics
 
 TODO
 
-#### Spikes
+## Spikes
 
-TODO
+In this section, we go over the procedure to follow when creating spikes. Exactly what
+these types of issues are and the steps to take when creating them are all outlined
+below. These guidelines apply for all projects falling underneath the
+[conda](github.com/conda) organization.
```

Comment:
```suggestion
[`conda`](github.com/conda) organization.
```

Reply:
According to our style guide, backticks are only necessary when talking about the command, not the org, package manager, etc.

Reply:
Thanks for the clarification, @kathatherine !